content/TrueCommand/Clusters.md

```diff
@@ -99,14 +99,15 @@ The SMB Network page has general options and fields added for each SCALE system
 |---------|-------|-------------|
 | NEXT | button | Continues to the next page. |
 | Skip this step | checkbox | Finalize the cluster configuration without registering a network to use for SMB sharing. Setting this permanently removes the ability to share data stored in this cluster. |
-| Interface | dropdown | SCALE system network interface to use for SMB traffic. For best functionality, choose an interface that can communicate between the individual SCALE systems, TrueCommand environment, Active Directory, and and client systems that need to access data shared from this cluster. |
+| Interface | dropdown | SCALE system network interface to use for SMB traffic. For best functionality, choose an interface that can communicate between the individual SCALE systems, TrueCommand environment, Active Directory, and client systems that need to access data shared from this cluster. |
```

Comment:
```suggestion:
Check if new style for TC involves deleting the second table column labeled "dropdown" and merging this information into the description so that the table only has 2 fields, Setting and Description (omit Value).

| Interface | dropdown | SCALE system network interface to use for SMB traffic. For best functionality, choose an interface that can communicate between the individual SCALE systems, TrueCommand environment, Active Directory, and client systems that need to access data shared from this cluster. |
```